includes/helper/class-wp-job-manager-helper.php

```diff
@@ -155,7 +155,7 @@ private function handle_admin_request() {
 		$product_slug = isset( $_GET['dismiss-wpjm-licence-notice'] ) ? sanitize_text_field( wp_unslash( $_GET['dismiss-wpjm-licence-notice'] ) ) : false;
 
 		if ( ! empty( $product_slug ) ) {
-			$product_plugins = $this->get_installed_plugins();
+			$product_plugins = $this->get_installed_plugins( false, false );
```

Comment:
This is a change in the current behavior. It seems like there was a bug that only allowed people to dismiss notices for _active_ plugins.